src/styles/blocks/card.scss

```
@@ -0,0 +1,150 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  grid-auto-flow: row;
+  justify-content: center;
+  align-items: center;
+  column-gap: 48px;
+  row-gap: 46px;
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  background: #fff;
+  padding: 32px 16px 16px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &__iMac {
+    box-sizing: border-box;
+    display: block;
+    height: 134px;
+    margin-left: auto;
+    margin-right: auto;
+    align-items: center;
+    justify-content: center;
+    border: none;
+  }
+
+  &__title {
+    display: flex;
+    align-items: center;
+    font-size: 12px;
+    font-family: inherit;
+    font-weight: 500;
+    margin-top: 40px;
+    color: $color-main;
+    line-height: 18px;
+    margin-bottom: 0;
+  }
+
+  &:hover .card__title {
+    color: #34568b;
+  }
+
+  &__code {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    font-family: inherit;
+    font-size: 10px;
+    color: $color-secondary;
+    margin-top: 4px;
+    line-height: 14px;
+    margin-bottom: 0;
+  }
+
+  &__ratings {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    color: $color-main;
+    font-size: 10px;
+    align-items: center;
+    height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+    justify-content: space-between;
+    height: 18px;
+
+    &--word {
+      display: flex;
+      padding: 0;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 400;
+      color: $color-secondary;
+      border: none;
+    }
+
+    &--value {
+      text-align: right;
+      padding: 0;
+      display: flex;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $color-main;
+      font-family: inherit;
+      border: none;
+    }
+  }
+
+  &__button {
+    display: flex;
+    height: 40px;
+    align-items: center;
+    justify-content: center;
+    border-radius: 5px;
+    background: $color-blue-accent;
+    margin-top: 16px;
+    border: 1px solid $color-blue-accent;
+    color: #fff;
+    text-transform: uppercase;
+    font-family: inherit;
+    font-weight: 700;
+    line-height: 16px;
+    font-size: 14px;
+    transition-duration: 300ms;
+  }
+
+  &__button:hover {
+    background: #fff;
+    color: $color-blue-accent;
+  }
```

Use scss nesting everywhere
```suggestion
  &__button {
    display: flex;
    height: 40px;
    align-items: center;
    justify-content: center;
    border-radius: 5px;
    background: $color-blue-accent;
    margin-top: 16px;
    border: 1px solid $color-blue-accent;
    color: #fff;
    text-transform: uppercase;
    font-family: inherit;
    font-weight: 700;
    line-height: 16px;
    font-size: 14px;
    transition-duration: 300ms;
    
    &:hover {
      // write styles here
    }
  }
```